### PR TITLE
1257620 update company document upload client

### PR DESCRIPTION
**Pull request by deviprasadkotian**
Updated client for company document upload client

### CODE REVIEW

**Review by Araballi — APPROVED** [review]
LGTM

**Review by Araballi — CHANGES_REQUESTED** [review]
version change

**Comment by NakulSeerpu** [comment]
Pls increment the number in directoru.build.props and also update  changes.md

File: Directory.Build.props [hunk 1]
```diff
@@ -19,7 +19,7 @@
 		<RepositoryUrl>https://github.com/kmdlogic/kmd-logic-citizen-documents-client</RepositoryUrl>
 		<RepositoryType>git</RepositoryType>
 		<GenerateAssemblyVersionAttribute>true</GenerateAssemblyVersionAttribute>
-		<VersionPrefix Condition=" '$(VersionPrefix)' == '' ">3.0.1</VersionPrefix>
+		<VersionPrefix Condition=" '$(VersionPrefix)' == '' ">3.0.2</VersionPrefix>
```
Comment (Araballi) [inline Directory.Build.props]: Asked to change 3.1.0

**Review by Araballi — APPROVED** [review]

**Pull request merged**